tests/test_api/test_other_zendesk_objects.py

```diff
@@ -428,3 +429,17 @@ def test_users_me(self):
             me = self.zenpy_client.users.me()
             self.assertNotEqual(me, None, "me is valid")
             self.assertNotEqual(me.email, "", "email is valid in me")
+
+class TestPasswordDeprecation(ZenpyApiTestCase):
+    __test__ = True
+    def test_password_failure(self):
+        with self.assertRaises(ZenpyException):
+            zenpy_client = Zenpy(subdomain="party", email="face@toe", password="Yer", password_treatment_level="error")
+            self.assertTrue(False)
+    def test_password_passes(self):
+        zenpy_client = Zenpy(subdomain="party", email="face@toe", password="Yer", password_treatment_level="warning")
+        self.assertTrue(True)
```

Comment:
Does this assertion accomplish anything? Is it possible to assert on the logging instead?

Reply:
I'll have to see if there's a mock or something that ensures the logger emitted the warning....that was the *RIGHT* way to do it.  I'll fix that.

Reply:
found, fixed.